scripts/package.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) 2019
+# Mainflux
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+# Collects EdgeX Go binaries (must be previously built)
+# and packages them into a dir alongside with necessary config files.
+# This facililtets transfer of binaries to otehr machine
+# (for example copy of cross-compiled binaris to ARM gateway)
+
+DIR=$PWD
+CMD_DIR=../cmd
+SERVICES=(config-seed export-client core-metadata core-command support-logging \
+  support-notifications sys-mgmt-executor sys-mgmt-agent support-scheduler \
+  core-data export-distro)
+
+PKG_DIR=${1:-edgex}
+
+# Copy binaries and config
+mkdir -p $PKG_DIR
+for i in "${SERVICES[@]}"; do
+  mkdir -p $PKG_DIR/$i
+  cp $CMD_DIR/$i/$i $PKG_DIR/$i
+  cp $CMD_DIR/$i/res/configuration.toml $PKG_DIR/$i 2>/dev/null || :
```

Comment:
this also doesn't copy over all of the properties files for config-seed for example

Reply:
I do not know well this `config-seed`, it is not clear completely to me how these https://github.com/edgexfoundry/edgex-go/tree/master/cmd/config-seed/res/properties are loaded. I see some Java configs. Additionally, I think this is Consul related - do we really need Consul in the single-instance on-gateway deployment scenario, where there is no service discovery and all the configs can be set manually for a given binary? I would say no...

Reply:
Yes, config-seed is used with consul, though in the future it will probably support other registries but today the only implementation of the registry abstraction is Consul. 

I don't think we should be making assumptions about how folks will deploy EdgeX even if they are running the binaries locally. If this was simply a packaging script used for your own purposes in a different repo sure that's fine to leave out `config-seed`, but since this is in upstream edgex-go and could have many possible deployment scenarios we should include all of what is "default" EdgeX even if a particular deployment may not use it (and they can always delete `config-seed` before doing a deployment - and note that this script doesn't even produce a deployment archive it just produces a directory with the files).

I'm more concerned about leaving out default functionality than I am with how many files are copied into a directory when running `scripts/package.sh`. If you feel strongly about needing to leave out config-seed, I'd instead like to see it implemented as an option to the script. For example, maybe some deployments don't need `support-notifications`, so having a generic option to leave out certain services seems more useful than just assuming that one service won't be used.